src/transforms/geoip.rs

```diff
@@ -0,0 +1,116 @@
+extern crate maxminddb;
```

Comment:
You can actually leave this `extern crate` out completely since we're fully on the 2018 edition.